tests/acceptance/features/apiGraph/unassignRole.feature

```diff
@@ -0,0 +1,96 @@
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Admin" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    # issue-5938 is related to below step, remove the comment when the issue is fixed
+    And user "Brian" should not be able to create a group "mygroup"
+    And user "Brian" should have the role "User" assigned
+
+  @issue-5938
+  Scenario: admin user unassign the role of another user with space admin role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Space Admin" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    # issue-5938 is related to below step, remove the comment when the issue is fixed
+    And user "Brian" should not be able to create a space "Project Mars" of type "project"
+    And user "Brian" should have the role "User" assigned
+
+
+  Scenario: admin user unassign the role of another user with user role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "User" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    When user "Brian" uploads file with content "uploaded content" to "testFile.txt" using the WebDAV API
+    Then user "Brian" should have the role "User" assigned
+
+
+  Scenario: admin user unassign the role of another user with guest role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Guest" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    When user "Brian" uploads file with content "uploaded content" to "testFile.txt" using the WebDAV API
+    Then user "Brian" should have the role "User" assigned
```

Comment:
Will the role of the user Brian be `user` only after uploading a file? Or is it something like this?
```suggestion
    Then the HTTP status code should be "204"
    And user "Brian" should not have any app role assigned
    But user "Brian" should have the user role "User" assigned
    When user "Brian" uploads file with content "uploaded content" to "testFile.txt" using the WebDAV API
    Then the HTTP status code should be "201"
```